workspaces/tech-insights/plugins/tech-insights-backend/package.json

```diff
@@ -47,9 +47,11 @@
   "dependencies": {
     "@backstage-community/plugin-tech-insights-common": "workspace:^",
     "@backstage-community/plugin-tech-insights-node": "workspace:^",
-    "@backstage/backend-common": "^0.22.0",
-    "@backstage/backend-plugin-api": "^0.6.18",
-    "@backstage/backend-tasks": "^0.5.23",
+    "@backstage/backend-common": "^0.23.3",
+    "@backstage/backend-defaults": "^0.4.1",
+    "@backstage/backend-plugin-api": "^0.7.0",
+    "@backstage/backend-tasks": "^0.5.27",
+    "@backstage/backend-test-utils": "^0.4.4",
```

Comment:
This was moved to a dependency.

Reply:
I'll try to get this written up as a bug.

Reply:
This has been on purpose to keep the public API intact. 

There is now a PR which changes removes that dependency 
https://github.com/backstage/community-plugins/pull/787